resources/sample_vaults/Tasks-Demo/.obsidian/plugins/obsidian-tasks-plugin/styles.css

```diff
@@ -13,11 +13,23 @@
     width: 1em;
     height: 1em;
     vertical-align: middle;
-    margin-left: 0.3em;
+    margin-left: 0.5em;
     cursor: pointer;
 }
 
-.internal-link.internal-link-short-mode {
+/* Urgency score */
+.tasks-score {
+    font-size: var(--font-ui-smaller);
+    font-family: var(--font-interface);
+    padding: 2px 6px;
+    border-radius: var(--radius-s);
+    color: var(--text-normal);
+    background-color: var(--background-secondary);
+    margin-left: 0.5em;
+    line-height: 1;
+}
+
+a.internal-link.internal-link-short-mode {
```

Comment:
Ah, I missed this diff several times in previous reading of the changes... It got buried in the diff, because it was adjacent to the added text - I only just noticed it.

```diff
- .internal-link.internal-link-short-mode {
+ a.internal-link.internal-link-short-mode {
```

For info, it's what I would like to talk about when we next chat... (no need to reply now)

Reply:
That change is going to be reverted... It turns out not to be necessary. (To test, make sure there is no underline on the chain-link icon for backlink in short mode.

Reply:
Now reverted, in both copies of the styles.css file